list-drafts.js

```diff
@@ -210,6 +210,12 @@ function displayDraftsTable(drafts) {
     }
     
     console.log(bottomBorder);
+    
+    // Display clickable file paths
```

Comment:
Replace the first column named Filename by the clickable file path of the post. And remove the Permalink column.